### PR TITLE
Bambi fix shape issues

### PR DESCRIPTION
**Pull request by AlexanderFengler**
fixes failing tests in #522 .

### CODE REVIEW

File: src/hssm/hssm.py [hunk 1]
```diff
@@ -975,27 +975,39 @@ def plot_trace(
...
-                        raise ValueError(
-                            "`var_names` must be a string, a list of strings, or None."
-                        )
-                else:
-                    kwargs["var_names"] = var_names
+                        kwargs["var_names"] = var_names
+        elif (not isinstance(data, az.InferenceData)) and (data is not None):
+            raise ValueError("data must be an InferenceData object.")
+        elif data is None:
+            raise ValueError("Please sample to model first.")
```
Comment (digicosmos86) [inline src/hssm/hssm.py:1010]: For clarity, I'd use just one if block. We already have an assert statement to ensure that `data` is of type `az.InferenceData`, so just an else block that throws an error should be sufficient

**Review by digicosmos86 — APPROVED** [review]
Looks great! Just one minor issue with if statements lol

**Comment by digicosmos86** [comment]
> Looks great! Just one minor issue with if statements lol

@AlexanderFengler also I am changing the merge target to the previous PR (fix InferenceData). We need to merge this one first before merging that one

File: src/hssm/hssm.py (hunk 1 above)
Comment (AlexanderFengler) [reply src/hssm/hssm.py]: Ok thinking about the logic, I think we were able to just take the whole if logic out and rely on the fact that `data` must be `InferenceData` at this point anyways. @digicosmos86

Comment (AlexanderFengler) [reply src/hssm/hssm.py]: This needed to change in two places.
Will let the tests run now, then merge?

**Pull request merged**